dot/core/helpers_test.go

```diff
@@ -17,10 +19,168 @@ import (
 	rtstorage "github.com/ChainSafe/gossamer/lib/runtime/storage"
 	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
 	"github.com/ChainSafe/gossamer/lib/utils"
+	"github.com/ChainSafe/gossamer/pkg/scale"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
+func balanceKey(t *testing.T, pub []byte) (bKey []byte) {
+	t.Helper()
+
+	h0, err := common.Twox128Hash([]byte("System"))
+	require.NoError(t, err)
+	h1, err := common.Twox128Hash([]byte("Account"))
+	require.NoError(t, err)
+	h2, err := common.Blake2b128(pub)
+	require.NoError(t, err)
+	bytes := [][]byte{h0, h1, h2, pub}
+	return concatenateByteSlices(bytes)
+}
+
+// Note might need to pass in Config but will see
+// CreateTestService is a new helper function to clean up orchestration for testing the txnPool
+func CreateTestService(t *testing.T, genesisFilePath string,
```

Comment:
nit Although it's unexported since it's in a test file, change this to a lowercase to avoid any confusion about exports